services/remotecluster/ping.go

```diff
@@ -99,6 +110,15 @@ func (rcs *Service) pingRemote(rc *model.RemoteCluster) error {
 		return err
 	}
 
+	if err := rcs.server.GetStore().RemoteCluster().SetLastPingAt(rc.RemoteId); err != nil {
+		rcs.server.GetLogger().Log(mlog.LvlRemoteClusterServiceError, "Failed to update LastPingAt for remote cluster",
+			mlog.String("remote", rc.DisplayName),
+			mlog.String("remoteId", rc.RemoteId),
+			mlog.Err(err),
+		)
+	}
+	rc.LastPingAt = model.GetMillis()
```

Comment:
Are we okay with setting this even if the store call failed?

Reply:
Setting that allows the state change event to be fired further up the call stack.  Worst case is we may have redundant events fired as next ping cycle will use the stale value from DB. It will correct itself once writes to that table start working again.